plexupdate.sh

```diff
@@ -548,7 +548,7 @@ if [ ! -z "${PLEXSERVER}" -a "${AUTOINSTALL}" = "yes" ]; then
 fi
 
 if [ "${AUTOINSTALL}" = "yes" ]; then
-	if ! hash ldconfig 2>/dev/null && [ -x /sbin/ldconfig -a "${DISTRO}" = "ubuntu" ]; then
+	if ! hash ldconfig 2>/dev/null && [ -a "${DISTRO}" = "ubuntu" ]; then
```

Comment:
Whoops, the -a is a typo, fixed.

The check for Ubuntu is because the thing that's failing is dpkg. Since RH doesn't use dpkg AFAIK, the fix isn't needed there so we avoid tampering with the PATH. If RH is also failing through some other mechanism we can fix that here as well, I just don't know if that's the case or not.